lib/IO/Socket/SSL.pm6

```diff
@@ -11,7 +11,6 @@ sub v6-split($uri) {
     $host ?? ($host, $port) !! $uri;
 }
 
-has Str $.encoding = 'utf8';
```

Comment:
This was removed because otherwise applying the role would die with `Attribute '$!encoding' already exists in the class 'IO::Socket::SSL', but a role also wishes to compose it`. However, it should be safe to remove, since IO::Socket provides the same default: https://github.com/rakudo/rakudo/blob/master/src/core.c/IO/Socket.pm6#L3